.github/pull_request_template.md

```diff
@@ -1,7 +1,14 @@
 ### Description
 
-[Summary of the changes]
+-
 
-### Modifications
+### Requirements
 
--
+- [ ] Guidelines followed
+- [ ] Documentation updated
+
+### Tests 🧪
+
+- [ ] Robot
+- [ ] [Container]()
```

Comment:
Empty link

Reply:
That's right. This was aimed to link to the docker image (if built), so it must stay empty in the template anyway.